gulpfile.js

```diff
@@ -1055,34 +1055,34 @@ gulp.task('publish', ['generic'], function (done) {
     });
 });
 
-gulp.task('test', function () {
+gulp.task('test', ['generic'], function () {
   return streamqueue({ objectMode: true },
     createTestSource('unit'), createTestSource('browser'));
 });
 
-gulp.task('bottest', function () {
+gulp.task('bottest', ['generic'], function () {
   return streamqueue({ objectMode: true },
     createTestSource('unit'), createTestSource('font'),
     createTestSource('browser (no reftest)'));
 });
 
-gulp.task('browsertest', function () {
+gulp.task('browsertest', ['generic'], function () {
   return createTestSource('browser');
 });
 
-gulp.task('unittest', function () {
+gulp.task('unittest', ['generic'], function () {
   return createTestSource('unit');
 });
 
-gulp.task('fonttest', function () {
+gulp.task('fonttest', ['generic'], function () {
```

Comment:
Unless you also update `test/font/jasmine-boot.js`, I don't think this change is necessary!?